README.md

```diff
@@ -66,79 +66,6 @@ More details on available commands can be found by running:
 make
 ```
 
-# Legacy developer set up instructions
```

Comment:
You think these are no longer useful at all? I've been doing all dev in docker, though it does make a few things - e.g. debugging - a bit more painful. It does seem nice having just one way to do things and focusing on documenting/improving that though. So actually I think I agree with your decision (just had to talk myself into it)

Reply:
LOL. Cool.

I always tell myself that I can use `git show` to see what code/docs used to look like. I really like the simplicity of the docker approach, but if we want to put back instructions to run it natively, we can change our minds.